care/facility/api/serializers/patient.py

```diff
@@ -218,6 +218,12 @@ class Meta:
     )
     abha_number_object = AbhaNumberSerializer(source="abha_number", read_only=True)
 
+    date_of_birth = serializers.DateField(required=False, allow_null=True)
+
+    is_antenatal = serializers.BooleanField(default=False)
+
```

Comment:
```suggestion
```

Reply:
@sainak if we remove this , it is throwing 500 error if "is_antenatal" is not passed in request body